c2corg_ui/templates/base.html

```diff
@@ -87,6 +91,15 @@
     <script src="${request.static_url('%s/moment/moment.js' % node_modules_path)}"></script>
     <script src="${request.static_url('%s/angular-moment/angular-moment.js' % node_modules_path)}"></script>
     <script src="${request.static_url('%s/angular-recaptcha/release/angular-recaptcha.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/ng-file-upload/dist/ng-file-upload.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/blueimp-load-image/js/load-image.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/slick-carousel/slick/slick.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/blueimp-load-image/js/load-image-orientation.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/blueimp-load-image/js/load-image-meta.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/blueimp-load-image/js/load-image-exif.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/blueimp-load-image/js/load-image-exif-map.js' % node_modules_path)}"></script>
```

Comment:
Not of what I would be aware. The min version uses all-in-one, the not minified versions are separated.